onnxruntime/core/providers/openvino/backend_manager.cc

```diff
@@ -118,9 +122,10 @@ bool BackendManager::ModelHasBatchedInputs(const ONNX_NAMESPACE::ModelProto& mod
   return has_batched_inputs;
 }
 
+#if 0
 #ifndef NDEBUG
 //Save ONNX Model
-static common::Status SaveModel(ONNX_NAMESPACE::ModelProto& model_proto,
+static common::Status SaveModel(Provider_ModelProto& model_proto,
```

Comment:
this was a very useful debug capability.

Reply:
Oops, I forgot to delete that block entirely. I simplified it to just the two lines in BackendManager::GetModelProtoFromFusedNode that originally called it. We have this type of code all over and this was the simplest one to do from the shared provider side.

Will push a new version in a minute.